README.md

```diff
@@ -277,3 +277,85 @@ importing and reuse.
+|             26   | Yes                  |                                                                                                 |
+|             27   | Yes                  |                                                                                                 |
+|             28   | Yes                  |                                                                                                 |
+|             29   | Yes                  |                                                                                                 |
+|             30   | Yes                  |                                                                                                 |
+|             31   | Yes                  |                                                                                                 |
+
+ 
+
+## ChEMBL Downloader Status
+
+ 
+
+|   ChEMBL Version | Downloader working   | Error                                                                                                                                        |
+|------------------|----------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+|               31 | Yes                  |                                                                                                                                              |
+|               30 | Yes                  |                                                                                                                                              |
+|               29 | Yes                  |                                                                                                                                              |
+|               28 | Yes                  |                                                                                                                                              |
+|               27 | Yes                  |                                                                                                                                              |
+|               26 | Yes                  |                                                                                                                                              |
+|               25 | Yes                  |                                                                                                                                              |
+|             24_1 | No                   | could not find chembl_24_1_sqlite.tar.gz in data for ChEMBL 24_1 in https://ftp.ebi.ac.uk/pub/databases/chembl/ChEMBLdb/releases/chembl_24_1 |
+|               24 | No                   | could not find chembl_24_sqlite.tar.gz in data for ChEMBL 24 in https://ftp.ebi.ac.uk/pub/databases/chembl/ChEMBLdb/releases/chembl_24       |
```

Comment:
this still seems to be incorrect, https://ftp.ebi.ac.uk/pub/databases/chembl/ChEMBLdb/releases/chembl_24/archived/chembl_24_sqlite.tar.gz is available.

Reply:
Do you propose changing the FTP path for versions as well?